### PR TITLE
Corrected the code

### PR DESCRIPTION
**Pull request by i-am-ebere**
It is important to pay attention to the details on your code.

### CODE REVIEW

**Review by joeymc406 — COMMENTED** [review]
i understand the change

**Review by joeymc406 — COMMENTED** [review]
what is the purpose of the classname movie card mb-3 i do not understand the mb-3 part

**Review by joeymc406 — COMMENTED** [review]
what is the purpose of the mb-3 in the classname

**Review by joeymc406 — COMMENTED** [review]
i understand my mistake i closed the button before it should have been closed and also the bracket mistake

File: src/components/movie-card/movie-card.jsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import "../../index.scss";
 
 export class MovieCard extends React.Component {
   render() {
     const { movie, onMovieClick } = this.props;
     
-    return <div className="movie-card" onClick={() => { onMovieClick(movie);
-    }}>{movie.Title}</div>;
+    return (
+    <div className="movie-card mb-3" onClick={() => onMovieClick(movie)}>
```
Comment (i-am-ebere) [inline src/components/movie-card/movie-card.jsx:9]: @joeymc406  look at the usage here and run the code on your machine to see

**Pull request merged**